Implement UserProfile

Doemuu opened the pull request:
Its 3 am I am so done with this.

Doemuu committed:
Implement user profile

Dan6erbond committed:
Update styling of user profile screen

 - Increase borders and padding.
 - Reduce usage of main color.
 - Convert featured submissions to cards for large screens.

Dan6erbond committed:
Fix signin link in header

Dan6erbond reviewed (approved):
Changes look great, the detailed profile statistics and content overview shows potential for use of the forum as a programming hub for IMS students.

I've made some additional adjustments to add more spacing and reduce the usage of our the green primary color which I find is still quite prominent on the page at this point. The header may need to be reworked to a simple dark header similar to GitHub's in order to retain focus on the main components of the site.

Dan6erbond committed:
Update layout styling

Make sidebar wider, header dark background and footer border in place of background
